website/pages/blog/the-guild-achieves-soc2-certification.mdx

```diff
@@ -0,0 +1,92 @@
+[Vanta](https://vanta.com/) compliance platform.
+
+As The Guild celebrates its 10th year of operation, this certification marks our transition from a
+trusted technology provider to an enterprise-ready partner equipped to meet the rigorous security
+and compliance requirements of organizations of all sizes.
+
+## What's SOC-2 Compliance? Why Type II Matters for Enterprises?
+
+[SOC-2](https://www.vanta.com/collection/soc-2/what-is-soc-2) (Service Organization Control 2) is a
+voluntary compliance standard developed by the American Institute of CPAs (AICPA) that specifies how
+organizations should manage customer data based on five "trust service criteria": security,
+availability, processing integrity, confidentiality, and privacy.
+
+While SOC-2 Type I certifications assess whether a company's systems are suitably designed at a
+specific point in time, **Type II certification** goes much further. It evaluates the operational
+effectiveness of those controls over a sustained period (in our case, 3 months), providing assurance
+that a company not only has appropriate security measures in place but consistently follows them.
+
+For enterprises, this distinction is crucial:
+
+1. **Proven Reliability**: Type II demonstrates consistent adherence to security practices over
+   time, not just a one-time assessment
+2. **Risk Reduction**: The certification significantly reduces the risk of data breaches and
+   security incidents
+3. **Compliance Support**: It helps enterprises meet their own regulatory and internal compliance
+   requirements
+4. **Vendor Management**: It simplifies vendor assessment processes, allowing for faster procurement
+   decisions
+5. **Trust Verification**: It provides independent verification of security claims by a qualified
+   third party
+
+## What This Means for The Guild Customers
+
+For our customers, this certification brings several tangible benefits:
+
+### Enhanced Security and Privacy
+
+Our SOC-2 Type II certification confirms that we have robust controls in place to protect your data.
+This includes comprehensive security policies, regular testing, monitoring, and incident response
+procedures that have been independently verified.
+
+### Simplified Vendor Assessment
+
+For enterprise customers with stringent security requirements, our certification streamlines the
+procurement process. Instead of conducting extensive security assessments, you can rely on our SOC-2
+Type II report as evidence of our security posture.
+
+### Enterprise Readiness
+
+This certification demonstrates that The Guild is an organization that can confidently serve
+enterprise customers with complex security and compliance requirements, while maintaining technical
+excellence.
+
+---
+
+<Callout emoji="🙏" type="info">
+
+We'd like to thank our team for their dedication to this process, our auditors at Advantage Partners
+for their thorough assessment, and the Vanta platform for streamlining our compliance journey.
```

Comment:
Vanta was really super helpful right?
should we add a few more words on how valuable they were and also how they help us keep the quality on an ongoing basis?